lib/components/Input/Input.scss

```
@@ -59,10 +55,10 @@ $border: 1px solid;
         &:focus,
         &:not(:placeholder-shown) {
             & ~ label {
-                font-size: 10px;
+                font-size: 1rem;
```

```suggestion
                font-size: 1rem;
```
var(--text-size-xs)?